app/controllers/sessions_controller.rb

```diff
@@ -19,4 +18,11 @@ def destroy
     redirect_to root_path
   end
 
+  private
+
+   def _log_in(user)
+    log_in user
```

Comment:
__[rubocop]__
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)